backend/src/test/java/heartbeat/service/report/GenerateReporterServiceTest.java

```diff
@@ -715,12 +720,24 @@ public void shouldNotGenerateCsvForPipeline() throws IOException {
 		Files.deleteIfExists(mockPipelineCsvPath);
 	}
 
+	@Test
+	public void shouldThrowNotFoundExceptionWhenExportCsvExpire() throws IOException {
+		ExportCSVRequest mockExportCSVRequest = ExportCSVRequest.builder()
+			.dataType("pipeline")
+			.csvTimeStamp("1685010080107")
+			.build();
+
+		assertThatThrownBy(() -> generateReporterService.fetchCSVData(mockExportCSVRequest))
+			.isInstanceOf(NotFoundException.class)
+			.hasMessageContaining("csv not found");
+	}
+
 	@Test
 	public void shouldReturnCsvDataForPipelineWhenExportCsv() throws IOException {
 		String mockedCsvData = "csv data";
 		ExportCSVRequest mockExportCSVRequest = ExportCSVRequest.builder()
 			.dataType("pipeline")
-			.csvTimeStamp("1685010080107")
+			.csvTimeStamp(String.valueOf(System.currentTimeMillis()))
```

Comment:
Should not use current datetime or coming datetime, 
PLZ using a previous datetime